types/poool-access/index.d.ts

```diff
@@ -664,8 +664,8 @@ export namespace Poool {
          *
          * ℹ️ Your Dashboard configuration will override these configuration values unless they have been set with a read-only mode.
          *
-         * @param {AccessConfigOptions} config - the configuration object
-         * @param {boolean} [readonly=false] - if true, the configuration will be read-only and cannot be overridden by the Dashboard
+         * - config - the configuration object
+         * - readonly - if true, the configuration will be read-only and cannot be overridden by the Dashboard
```

Comment:
The correct form of this would be:

```
         * @param config - the configuration object
         * @param [readonly=false] - if true, the configuration will be read-only and cannot be overridden by the Dashboard
```

You don't need to drop JSDoc format if you don't want to, just drop the types.